src/firewall/client.py

```diff
@@ -3124,6 +3124,10 @@ def config(self):
     def reload(self):
         self.fw.reload()
 
+    @handle_exceptions
+    def reset(self):
```

Comment:
Please call it `reset_to_defaults`, i.e. keep the naming consistent.

Reply:
Fixed, also updated reset() in server/firewalld.py.